impeller/entity/entity_unittests.cc

```diff
@@ -1931,238 +1931,6 @@ TEST_P(EntityTest, SrgbToLinearFilter) {
   ASSERT_TRUE(OpenPlaygroundHere(callback));
 }
 
-TEST_P(EntityTest, TTTBlendColor) {
```

Comment:
What's replacing these tests?

See also https://github.com/flutter/flutter/issues/128606. It's possible some of these tests are encoding bad behavior, but it seems veyr likely to me that if they are not encoding bad behavior they're missing cases where there's bad behavior.

Reply:
Sorry, I see below the new tests.

Reply:
Yeah I just moved this existing test into `GeometryTest.ColorBlend`. And yes, some of them are wrong -- we used to use these for absorbing the DrawPaints, which introduced bugs. Some blends are also missing from the test at the moment.